### PR TITLE
0.1.3 candidate

### PR DESCRIPTION
**Pull request by LT-Rascek**
Dev-Only comments are in square brackets (and shouldn't be included in patch notes)

Faiths:
* Fixup some Hellenic gender laws vis-a-vis Pythagoreanism and Platonism
* Assyrian Pagans gain piety from raiding in addition to other effects
   * Done as a separate event/notification for better long-term vanilla compatibility
* Audianism, Patripassianism use Communion instead of Aniconism as they have a HOF
* Sethians use Communion instead of Inner Journey because Inner Journey isn't available to Dualists
* Hepthalites use Lay Clergy
* Made Slavic pagan pluralist instead of fundamentalist
* [Removed only slightly-modified pagan faiths for greater Vanilla compatibility; utilizing game_start on actions to set doctrines appropriately]

Cultures:
* Przeworsk become Cosmopolitan (i.e., Egalitarian) to better reflect contact culture status and make amenable with Vistula-Veneti divergence
* Old Slavic touch ups (Stoic instead of Bellicose + Collective Lands)
* Fixup Vidivarii ethnicities

Localization:
* Hypsisterian holy book changed to match Hellenism
* Fix Armenian Chancellor name
* Update tooltip as to why k_byznatium can't be made

Graphics:
* Change Valentinianism color for more clarity
* Ported in new religion icons, courtesy of super7700

Gameplay:
* Added empire creation rule to allow players to create multiple empires if they so desire (like in Vanilla)
* Modified Zeno's War CB to account for total occupation of Zeno's provinces somewhat.

Other:
* Updated to Vanilla 1.8.0

Bug Fixes:
* Fix Valentinianism consanguinity doctrine
* Issue with Italian kingdom triggers
* Missing localization for roman empire triggers
* Fix WRE de jure flips
* Removed hanging prefeudal conversion reference that was missed
* Fix create Israel decision

### CODE REVIEW

**Comment by LT-Rascek** [comment]
Notes for Bug Followup:
"Can't usurp Italy by faction war"
Landing title makes it possible to press for peace while unlanding makes it invalidate. Might be a deeper level issue with a claim CB, and we may requires separate CBs for titular and non-titular claims, as changes to `claim_cb` aren't cleanly registering.

File: WtWSMS/common/casus_belli_types/BP_event_war.txt [hunk 1]
```diff
@@ -736,11 +736,12 @@ zenos_claim_cb = { #Like a claim_cb, but has war-scoped special troops
...
 
-	attacker_wargoal_percentage = 0.0
+	attacker_wargoal_percentage = 0.0 #Ticking warscore when at least one of Basilcos' provinces occupied
+	defender_wargoal_percentage = 100.0 #Ticking warscore when every single one of Zeno's holdings occupied
 	max_attacker_score_from_battles = 100
 	full_occupation_by_defender_gives_victory = no
-	defender_score_from_occupation_scale = 0 #Occupation doesn't help the defender, only battles
-	max_defender_score_from_occupation = 0 #Occupation doesn't help the defender, only battles
+	defender_score_from_occupation_scale = 50
+	max_defender_score_from_occupation = 30 #You *really* need to defeat Zeno on the battlefield, not just occupy his provinces
```
Comment (loup99) [inline WtWSMS/common/casus_belli_types/BP_event_war.txt:744]: This should be added to the changelog.

File: WtWSMS/common/culture/cultures/BP_east_germanic.txt [hunk 2]
```diff
@@ -508,9 +508,11 @@ vidivarii = { # Migrational Era Vidivarii
 	unit_gfx = { norse_unit_gfx }
 	
 	ethnicities = {
-		33 = circumpolar_dark_hair
-		33 = circumpolar_brown_hair
-		33 = circumpolar_blonde_hair
+		#50/50 Vandal/Baltic Admixture (then added to avoid any code problems)
```
Comment (loup99) [inline WtWSMS/common/culture/cultures/BP_east_germanic.txt:511]: Source for this specific origin?

File: WtWSMS/common/factions/00_factions_BP_overwrite.txt [hunk 3]
```diff
@@ -313,6 +326,21 @@ liberty_faction = {
...
+			FACTION_TARGET = scope:target
+		}
+
+		custom_description = {
+			text = has_lowest_crown_authority
+			subject = scope:target
+			scope:target = { realm_law_supports_liberty_faction_trigger = yes }
+		}
+	}
+	##### Changed from Vanilla ^^^^^
```
Comment (loup99) [inline WtWSMS/common/factions/00_factions_BP_overwrite.txt:341]: Is this related to the Italy faction war bug?

File: WtWSMS/common/factions/00_nation_fracturing_faction_BP_overwrite.txt [hunk 4]
```diff
@@ -68,6 +68,23 @@ nation_fracturing_faction = {
...
+		common_can_character_create_trigger = {
+			FACTION_TARGET = scope:target
+		}
+
+		liege = {
+			is_independent_ruler = yes
+			NOT = { has_government = theocracy_government }
+		}
+
+		# Only powerfull vassals can create such factions and they need to have either another culture or faith
```
Comment (loup99) [inline WtWSMS/common/factions/00_nation_fracturing_faction_BP_overwrite.txt:81]: Small typo on "powerful".

File: WtWSMS/credits.txt [hunk 5]
```diff
@@ -11,11 +11,11 @@ Loup99
 Symmachus
 
 $TITLE_OTHER_MODS_PROVIDING_CONTENT$
-Vertimanus for the "Community Culture Utility"
-Successions Expanded Team
 Chalk and Rootel for "More Faith Icons"
-herkles (aka Vyshan) for Dharmachakra
 Elder Kings 2 Mod Team for their sub-bookmark system
```
Comment (loup99) [inline WtWSMS/credits.txt:15]: Do we still need those given the vanilla change?

File: WtWSMS/common/culture/cultures/BP_east_slavic.txt [hunk 6]
```diff
@@ -1,7 +1,12 @@
-﻿old_slavic = {
+﻿#Notes:
+#[1] - From what I've read of proto-Slavic groups, they were fairly settled and had, at most, a relatively
+# flat social structure, with social stratification occurring when population density increased. Collective
+# lands seems to be the "best fit" for this notion of a flat hierarchy
```
Comment (loup99) [inline WtWSMS/common/culture/cultures/BP_east_slavic.txt:4]: According to Peter Heather in Empires and Barbarians, it was during the 6th century that most Slavs underwent socio-political changes leading to an increasing stratification. Prior to that there are indeed no attested mentions of Slavic leaders in for example Procopius. So I don't know if we should add mechanics for this to evolve or if we should at start keep the previous setup since it was underway.

File: WtWSMS/common/casus_belli_types/BP_event_war.txt (hunk 1 above)
Comment (LT-Rascek) [reply WtWSMS/common/casus_belli_types/BP_event_war.txt:744]: Done.

File: WtWSMS/common/culture/cultures/BP_east_germanic.txt (hunk 2 above)
Comment (LT-Rascek) [reply WtWSMS/common/culture/cultures/BP_east_germanic.txt:511]: Bartosz Kontny's work on the Vidivarii. 
https://www.researchgate.net/publication/339977371_The_Making_of_the_Vidivarii_Germanic_and_Baltic_Interculturation_in_the_Late_5th_Century

File: WtWSMS/common/factions/00_factions_BP_overwrite.txt (hunk 3 above)
Comment (LT-Rascek) [reply WtWSMS/common/factions/00_factions_BP_overwrite.txt:341]: No, that's just a change in Vanilla for 1.8.*

File: WtWSMS/common/factions/00_nation_fracturing_faction_BP_overwrite.txt (hunk 4 above)
Comment (LT-Rascek) [reply WtWSMS/common/factions/00_nation_fracturing_faction_BP_overwrite.txt:81]: Vanilla typo in 1.8.1, actually.

File: WtWSMS/credits.txt (hunk 5 above)
Comment (LT-Rascek) [reply WtWSMS/credits.txt:15]: Not sure what you mean? This overwrites the Vanilla credits with the actual mod credits.

File: WtWSMS/common/culture/cultures/BP_east_slavic.txt (hunk 6 above)
Comment (LT-Rascek) [reply WtWSMS/common/culture/cultures/BP_east_slavic.txt:4]: I have no strong feelings one way or the other. But it strikes me as more of a 0.2.0 for more extensive mechanic changes like that. I think "Collective Lands" works as a stop-gap for 0.1.3 though.

Comment (loup99) [reply WtWSMS/common/culture/cultures/BP_east_slavic.txt:4]: Ok, fine by me to include this as 0.2.0 material, I would just add a comment to remember it given the general workload of that update 😉

File: WtWSMS/credits.txt (hunk 5 above)
Comment (loup99) [reply WtWSMS/credits.txt:15]: I'm referring to the credits to the Elder Kings 2 Mod Team for their sub-bookmark system. Does the new vanilla bookmark not overwrite it?

Comment (LT-Rascek) [reply WtWSMS/credits.txt:15]: Oh, right, yes. I'll remove that then.

File: ChangeLog.txt [hunk 7]
```diff
@@ -1,6 +1,46 @@
...
+
+Bug Fixes:
+- Fix Valentinianism consanguinity doctrine
+- Issue with Italian kingdom triggers
+- Missing localization for roman empire triggers
+- Fix WRE de jure flips
+- Removed hanging prefeudal conversion reference that was missed
+- Fix create Israel decision
+
 BUGFIXES:
```
Comment (loup99) [inline ChangeLog.txt:44]:
```suggestion
```

File: WtWSMS/common/culture/cultures/BP_east_germanic.txt (hunk 2 above)
Comment (loup99) [inline WtWSMS/common/culture/cultures/BP_east_germanic.txt:511]:
```suggestion
		#50/50 Gepid/Baltic Admixture (then added to avoid any code problems)
```

File: WtWSMS/common/culture/cultures/BP_east_slavic.txt (hunk 6 above)
Comment (LT-Rascek) [reply WtWSMS/common/culture/cultures/BP_east_slavic.txt:4]: Commit this comment in the Design notes for 0.2.0

Comment (loup99) [inline WtWSMS/common/culture/cultures/BP_east_slavic.txt:4]:
```suggestion
# lands seems to be the "best fit" for this notion of a flat hierarchy, review evolution for 0.2.0
```

**Review by loup99 — APPROVED** [review]
